src/main.ts

```diff
@@ -5,6 +5,9 @@ import {handlePrClosed} from "./pr_closed";
 import {TerraformCloudApi} from "./tfc_api";
```

Comment:
These changes will probably be easier to understand looking at the file directly rather than at this diff. Most of the changes are purely organizational to make the code easier to follow, except where code actually needed to be changed to accommodate the backend swap.